contracts/access/manager/AccessManager.sol

```diff
@@ -611,7 +613,7 @@ contract AccessManager is Context, Multicall, IAccessManager {
      */
     function _getAdminRestrictions(
         bytes calldata data
-    ) private view returns (bool restricted, uint64 roleAdminId, uint32 executionDelay) {
+    ) private view returns (bool adminRestricted, uint64 roleAdminId, uint32 executionDelay) {
         if (data.length < 4) {
             return (false, 0, 0);
```

Comment:
maybe we want to do
`return (false, getTargetFunctionRole(address(this), bytes4(0)), 0);`
so that selector 0 can be used to set restriction on the `receive()` function

Reply:
```suggestion
            return (false, getTargetFunctionRole(address(this), bytes4(0)), 0);
```

Reply:
We need to decide on this, but otherwize the PR is good to me.